### PR TITLE
Use secure links in README

### PR DESCRIPTION
**Pull request by sidrao2006**
Previously #254 

## Description

Use secure links in README to improve pub points

## Checklist

- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by sidrao2006** [comment]
cc @xvrh

File: generator/lib/builders/readme_builder.dart [hunk 1]
```diff
@@ -9,7 +9,7 @@ String buildReadmeMd(Api api) {
 **Warning: This is a generated library, some operations may not work.**
 
 *About the service:*
-${markdownText(api.documentation)}
+${markdownText(api.documentation.replaceFirst('http', 'https'))}
```
Comment (xvrh) [inline generator/lib/builders/readme_builder.dart:12]: `replaceAll('http://', 'https://')`.
Else you will replace "https://" to "httpss://"

**Comment by xvrh** [comment]
You can run the generator and see the result of your change with `dart bin/generate.dart generate --no-build` from the "generator" folder.

File: generator/lib/builders/readme_builder.dart (hunk 1 above)
Comment (sidrao2006) [reply generator/lib/builders/readme_builder.dart:12]: Right, sorry

**Comment by Schwusch** [comment]
I believe many links are probably fixed in newer versions of aws-sdk-js, the version we are using is [6 months old](https://www.npmjs.com/package/aws-sdk/v/2.659.0) at this time of writing. Perhaps we should look at bumping that version some time, but I am afraid it will bring a lot of work to catch up.

**Comment by Schwusch** [comment]
This is solved in the generator nowadays

**Pull request closed without merging**